src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +34,32 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              Todo #{id}
             </div>
 
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button type="button" className="delete" data-cy="modal-close" />
+            <button
+              type="button"
+              className="delete"
+              data-cy="modal-close"
+              onClick={() => onClose(null)}
```

Comment:
The naming of the 'onClose' function does not follow the naming conventions provided in the checklist. It would be better to rename it to 'handleClose' to be more descriptive and to follow the convention.

Reply:
ohhhhhhhhhhhhhh